lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
@ziuq one way to dig into commit history and git blame info is to play around with the github interface.
Start out by browsing to [Scripts/setup.pm Line 421](https://github.com/ziuq/LedgerSMB/blob/master/lib/LedgerSMB/Scripts/setup.pm#L421)
Then at the top of the page you can click on "blame" or "history"

Once there you can click on either a commit (in History) or a blame entry (on the left in Blame) to get more info.

You can do the same from the commandline, but sometimes it's easier to learn what's possible from a Visual interface.

There are also some GUI tools you can install on your local desktop that may help.
These are just 2 of many
- `Git Cola`
- `Git DAG`